packages/flutter_tools/lib/src/commands/doctor.dart

```diff
@@ -8,7 +8,9 @@ import '../doctor.dart';
 import '../runner/flutter_command.dart';
 
 class DoctorCommand extends FlutterCommand {
-  DoctorCommand() {
+  final bool verbose;
+
+  DoctorCommand({this.verbose: false}) {
```

Comment:
trivial nit: in flutter code we prefer to have the constructors first, that way it's trivial to see if there's an implicit constructor or not (since otherwise, you have to scan the entire class' definition to determine that).

Reply:
done